app/boards/arm/nrfmicro/nrfmicro_11.dts

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2020 Okke Formsma, joric
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+#include "arduino_pro_micro_pins.dtsi"
+
+/ {
+	model = "nrfmicro_flipped";
```

Comment:
Is this supposed to be "nrfmicro". Honestly I have no idea what this file is and how it works.
I need to stare at this codebase for some more time to understand more.

Reply:
Good catch, this should be "nrfmicro" @okke-formsma